cocos/platform/android/jni/JniCocosActivity.cpp

```diff
@@ -216,17 +216,17 @@ JNIEXPORT void JNICALL Java_com_cocos_lib_CocosActivity_onSurfaceCreatedNative(J
     setWindow(ANativeWindow_fromSurface(env, surface));
 }
 JNIEXPORT void JNICALL Java_com_cocos_lib_CocosActivity_onStartNative(JNIEnv *env, jobject obj) {
```

Comment:
Function name should be named liked this to be invoked by java codes. So do not have to fix it, but should fix other coding style issues.